scipy/ndimage/_morphology.py

```diff
@@ -202,6 +202,14 @@ def generate_binary_structure(rank, connectivity):
             [False,  True, False],
             [False, False, False]]], dtype=bool)
 
+    Further Examples
+    ----------------
```

Comment:
We cannot add another section in the doc. Only specific sections are allowed (like `Examples`). Here Just write a single line to explain the other example. Ideally link it to the previous ones.